### ISSUE TITLE
doi request fails - existing datacite.yml not found

### ISSUE THREAD

**Issue by mikapfl** [G-Node/gin-doi#62]
**Describe the bug**

If I click on the button "request doi" in the repository https://gin.g-node.org/mikapfl/read_di_unfccc , I get the error message:
```
DOI request failed

The DOI file is missing or not valid. See the messages below for specific issues with the provided data.
Also, please see the DOI guide for detailed instructions.

No datacite.yml file found in repository
```
However, the datacite.yml definitely exists: https://gin.g-node.org/mikapfl/read_di_unfccc/src/main/datacite.yml

The problem might be related (wild guess) to the fact that this repository has no "master" branch and uses a "main" branch instead.

**Comment by achilleas-k** [comment]
You're right, it must be because of the branch name.  I'll have to make it look into the default branch rather than assume it's master.

Thanks for the report.
